src/puppeteer_pool.js

```diff
@@ -21,9 +23,9 @@ const DEFAULT_OPTIONS = {
     maxOpenPagesPerInstance: 50,
     retireInstanceAfterRequestCount: 100,
 
-    // These can't be constants because we need it for unit tests.
-    instanceKillerIntervalMillis: 60 * 1000,
-    killInstanceAfterMillis: 5 * 60 * 1000,
+    puppeteerOperationTimeoutSecs: 15,
```

Comment:
I'd set this quite high as it's quite rare to minimize probability that the browser gets started after the timeout. Even 3 minutes would be fine.

Reply:
If there is handleRequestFunctionTimeout then we must fit this to that timeout but as mentioned above we could remove it and then we can set this high to minimize probability of line 269 (browser gets started after timeout)